vignettes/teal-options.Rmd

```diff
@@ -84,6 +84,10 @@ This indicates whether to print the `JavaScript` console logs to the `R` console
 
 Default: `FALSE`.
 
+### `teal.renv.lockfile` (`character`)
+
+The path to the pre-computed `renv` `.lockfile` that will be shared through teal app. To read more about `.lockfile` usage creation check `?teal::create_renv_lockfile`.
```

Comment:
`?teal::create_renv_lockfile` or `?teal:::create_renv_lockfile`?

Reply:
@pawelru now it's `teal::teal_lockfile`

Reply:
but will double check if we need `::` or `:::
`

Reply:
`teal_lockfile` is not exported, and kept as internal. It is not displayed in INDEX via `help(p = 'teal')`. 
However `?teal::teal_lockfile` and `?teal_lockfile` do open a page with information about this functionality. No need for `:::`

Reply:
OK thanks for checking!